pkg/strategy/drift/output.go

```diff
@@ -26,6 +22,16 @@ func (a jsonArr) Len() int           { return len(a) }
 func (a jsonArr) Less(i, j int) bool { return a[i].key < a[j].key }
 func (a jsonArr) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 
+func canInt(v reflect.Value) bool {
```

Comment:
Rename to `isInt` and move it to the pkg/dynamic package?

Reply:
canInt maps to https://pkg.go.dev/reflect#Value.CanInt

Reply:
got it